src/App.js

```diff
@@ -1,24 +1,18 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import AddView from './Components/AddView/addView';
+import ListView from './Components/ListView/listView';
+import './index';
 
 function App() {
```

Comment:
Since you are using arrow functions throwout your app, you can change this function to an arrow function too.